doc/ref/GalSim_Quick_Reference.tex

```diff
@@ -244,6 +244,11 @@ \subsection{GSObject classes and when to use them}\label{sect:gsobjectclasses}
     with index \texttt{n=1}.  Requires one of the
     following \texttt{\emph{size}} parameters: \texttt{scale\_radius} or \texttt{half\_light\_radius}.
 \item[$\circ$]
+  \texttt{galsim.}\href{http://galsim-developers.github.com/GalSim/classgalsim_1_1base_1_1_inclined_exponential.html}{\texttt{InclinedExponential}}\texttt{(inclination, scale\_radius, scale\_height, flux=1.)}
```

Comment:
URL does not exist, presumably because we only put up documentation for the last tagged version, which this is not in.  For now, I recommend simply not having a URL in the quick reference (rather than linking to something that doesn't exist).